exasol/slc/internal/tasks/upload/deploy_container_base_task.py

```diff
@@ -68,7 +67,8 @@ def run_task(self):
         lang_def_builder = LanguageDefinitionsBuilder(language_def_components_list)
         result = DeployResult(
             release_path=str(release_path),
-            upload_url=self._complete_url(export_info),
+            human_readable_upload_location=self._complete_url(export_info),
+            bucket_path=path_in_bucket,
```

Comment:
hm, if think about, this means we are serializing the credentials to a file, because that is how we return files.